MvvmCross/Binding/iOS/Target/MvxBaseUIDatePickerTargetBinding.cs

```diff
@@ -21,15 +23,18 @@ protected MvxBaseUIDatePickerTargetBinding(object target, PropertyInfo targetPro
             var datePicker = View;
             if (datePicker == null)
             {
-                MvxBindingTrace.Trace(MvxTraceLevel.Error,
-                                      "Error - UIDatePicker is null in MvxBaseUIDatePickerTargetBinding");
+                MvxBindingTrace.Trace(MvxTraceLevel.Error, "Error - UIDatePicker is null in MvxBaseUIDatePickerTargetBinding");
             }
-            else
+            else if (targetPropertyInfo.Name == nameof(UIDatePicker.Date)) // Only listen for changes if we are binding against the Date property.
```

Comment:
@DaRosenberg This line is breaking the target binding for `CountDownDuration` changes.

Reply:
Yep, I can see why. I'll fix.